uw-frame-static/superstatic.json

```diff
@@ -0,0 +1,10 @@
+{
+  "rewrites" : [
+    {"source":"/settings", "destination" : "index.html"},
+    {"source":"/notifications", "destination" : "index.html"},
+    {"source":"/features", "destination" : "index.html"},
+    {"source":"/about", "destination" : "index.html"},
+    {"source":"/server-error", "destination" : "index.html"},
+    {"source":"/access-denied", "destination" : "index.html"}
+  ]
+}
```

Comment:
:+1:
I think this is intriguing because it seems like it wouldn't be too hard to pull the rewrites and the routes from the same js/json, and prevent them getting out of sync.